### ISSUE TITLE
fix HHVM support

### ISSUE THREAD

**Issue by lsmith77** [liip/LiipTranslationBundle#29]

**Comment by aztech-dev** [comment]
So far, it looks like an issue with the crawler and session management.

Commenting out https://github.com/liip/LiipTranslationBundle/blob/master/Tests/Controller/ImportControllerTest.php#L122-L125 makes the test pass and value is correctly updated, so it seems that session content is lost after the redirect.

I dont know enough about DomCrawler to say as to why the redirect is dropping the updated values, I'll try to investigate further tonight.

**Comment by lsmith77** [comment]
@aztech-dev yeah .. I can confirm this. the data written to disk are the same. I also tried refactoring things to use another request to fetch the data, rather than going to the container directly and that doesn't change anything either.

**Comment by lsmith77** [comment]
@jeanmonod I wonder if we should not just remove this assertion so that we can mark HHVM as compatible in travis .. wdyt?

**Comment by jeanmonod** [comment]
If you think that this is a bug in the `DomCrawler` why not. Is anybody have tried to reproduce/test the import action in a real browser?
